packaging/installer/kickstart.sh

```diff
@@ -584,7 +584,7 @@ check_for_remote_file() {
   elif [ -n "${CURL}" ]; then
     "${CURL}" --output /dev/null --silent --head --fail "${url}" || return 1
   elif command -v wget > /dev/null 2>&1; then
-    wget -S --spider "${url}" 2>&1 | grep -q 'HTTP/1.1 200 OK' || return 1
+    wget -q --spider "${url}" || return 1
```

Comment:
@netdata/agent-sre Please confirm this is OK, if not we can minimise the risk by using the previous version always, except for the cases where `-S` is not available in `wget`:
```
wget -S --spider "${url}" 2>&1 | grep -q 'HTTP/1.1 200 OK' || wget -q --spider "${url}" || return 1
```

Reply:
removing `-S` changes the way this function works, we want just to check if remote files exists without the payload for any kind of reasons. I suppose since there is no way for openWRT package to execute our wget instruction with `-S`, use curl instead; to that end make curl a dependency. wget is the fallback after all